pkg/ui/package.json

```diff
@@ -104,13 +105,16 @@
     "stylus": "^0.54.5",
     "stylus-loader": "^3.0.1",
     "thread-loader": "^1.1.1",
+    "topojson": "^3.0.2",
     "ts-loader": "^2.3.4",
     "tslint": "^5.10.0",
     "typescript": "^2.2.1",
     "url-loader": "^0.5.8",
+    "us-atlas": "^1.0.2",
```

Comment:
Nice that these are available as node modules. If we're not doing US states, remove this one (but I prefer to do US states I think)